.github/workflows/build.yml

```diff
@@ -23,6 +23,12 @@ jobs:
         run: ./tests/do.sh
       - name: Test UNIT
         run: ./tests/unit/unit
+      - name: Configure (with debug logs)
+        run: make distclean && env CFLAGS='-Werror' ./autogen.sh --enable-debug-messages
+      - name: Build
+        run: make all
+      - name: Test DIFF
+        run: ./tests/do.sh
```

Comment:
Is it necessary to do the diff tests twice?

Reply:
Save ressources whenever possible. :)

Reply:
No strong opinion on that... I think that if we compile some more code, we should somehow check that it doesn't crash anything, at very least. But run the entire test suite might be a bit overwhelming...
Do you prefer to remove the tests anyway?

Reply:
I mean it is just a "print some pretty debug messages"-mode. There shouldn't be lot's of additional code paths to check. And `./tests/do.sh` is always sooo slow.

Reply:
Ok!